incubator/java-openliberty/templates/kafka/pom.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version='1.0' encoding='utf-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent><!--required parent POM-->
+        <groupId>dev.appsody</groupId>
+        <artifactId>java-openliberty</artifactId>
+        <version>{{.stack.parentpomrange}}</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>dev.appsody.starter.java-openliberty</groupId>
+    <artifactId>starter-app</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>2.3.0</version>
```

Comment:
We could follow-up and refactor these two: kafka-clients, and slf4j as provided dependencies.   It allows better layering of Docker image by building a thinner WAR.   But this should be fine for now, since at the moment, it's not trivial to copy the dependencies in the maven build (some follow-up here: https://github.com/OpenLiberty/ci.maven/issues/705)